src/Gateway.php

```diff
@@ -28,6 +28,16 @@ public function ready()
         return ($this->authorizedSupplier && $this->authorizedBuyer && $this->jwt);
     }
 
+    /**
+     * method to return data for front end analytics configuration
+     * @return array
+    */
+    public function getFrontendConfig($token)
```

Comment:
Suggest `getConfig()` as we could extend the functionality and that naming scheme maps more intuitively/cleanly to what's actually happening

Reply:
👍